### ISSUE TITLE
Bread created but can not access to new created BREAD

### ISSUE THREAD

**Issue by raymondugv** [thedevdojo/voyager#4712]
## Version information

 - Laravel: v6.11.0
 - Voyager: v1.3
 - PHP: 7.3.7
 - Database: MySQL 5.7.26 (e.g. MySQL 8.0)

## Description

I've created BREAD after creating a database as well as model. Then when click to create BREAD, the system said 404 | Error
![image](https://user-images.githubusercontent.com/21214764/72800873-e4c4c980-3c7a-11ea-99cb-b7b1ab6e526b.png)

Then after that, when checking again at BREAD index, it still shows that the BREAD has been created.
![image](https://user-images.githubusercontent.com/21214764/72800934-058d1f00-3c7b-11ea-938a-6eddf7eaef21.png)

But when trying to access BREAD, it said error 403
![image](https://user-images.githubusercontent.com/21214764/72800991-22295700-3c7b-11ea-9fe2-56a529323d54.png)

After giving it a permission
![image](https://user-images.githubusercontent.com/21214764/72801035-42591600-3c7b-11ea-86c5-7b78009bc267.png)
This is what I've seen on the screen
![image](https://user-images.githubusercontent.com/21214764/72801066-56047c80-3c7b-11ea-8507-4f3ce199a1bf.png)

But I've already created that database using Voyager

![image](https://user-images.githubusercontent.com/21214764/72801133-70d6f100-3c7b-11ea-8065-7eb08ba93927.png)

What am I missed? 😔 

## Steps To Reproduce
Steps to reproduce the behaviour:
1. Create booking_informations database and model
2. Create BREAD using Voyager Interface
3. Set permission to that BREAD

## Expected behaviour
- I should be able to access to Booking Information BREAD index

## Screenshots
Screenshots included on the error description

Thank you for a super great system!

**Comment by emptynick** [comment]
Looks like your naming is wrong.
The correct plural is information not informations and so Laravel is looking for that table without s.

**Comment by raymondugv** [comment]
> Looks like your naming is wrong.
> The correct plural is information not informations and so Laravel is looking for that table without s.

Thank you for responding, @emptynick 

You mean on the table name, right?

**Comment by MrCrayon** [comment]
Yes he means the table, if you solved please close the issue.

**Comment by raymondugv** [comment]
> Yes he means the table, if you solved please close the issue.

Thank you @MrCrayon ,

My problem is fixed, but when creating BREAD, my site always calls a 404 error, what happened?.
![image](https://user-images.githubusercontent.com/21214764/72803162-f78dcd00-3c7f-11ea-95b0-449b7ceb73d5.png)

Like this, do you facing that same issues that I did?

**Comment by MrCrayon** [comment]
Works for me.

What's your Voyager exact version v1.3.0 or v1.3.1?

Try clearing your cache:
```
php artisan view:clear
php artisan cache:clear
```

**Comment by raymondugv** [comment]
> Works for me.
> 
> What's your Voyager exact version v1.3.0 or v1.3.1?
> 
> Try clearing your cache:
> 
> ```
> php artisan view:clear
> php artisan cache:clear
> ```

Thank you for responding, 

My current Voyager version is 1.3.1.
And by the way, it still has the same issues after I've run above command.

**Comment by MrCrayon** [comment]
Does that happen only after saving or also when you go to Tools -> BREAD?

How are your routes for bread?

```
php artisan route:list | grep bread

|        | POST      | admin/bread                          | voyager.bread.store               | TCG\Voyager\Http\Controllers\VoyagerBreadController@store              | web,admin.user                                       |
|        | GET|HEAD  | admin/bread                          | voyager.bread.index               | TCG\Voyager\Http\Controllers\VoyagerBreadController@index              | web,admin.user                                       |
|        | GET|HEAD  | admin/bread/delete_relationship/{id} | voyager.bread.delete_relationship | TCG\Voyager\Http\Controllers\VoyagerBreadController@deleteRelationship | web,admin.user                                       |
|        | POST      | admin/bread/relationship             | voyager.bread.relationship        | TCG\Voyager\Http\Controllers\VoyagerBreadController@addRelationship    | web,admin.user                                       |
|        | PUT       | admin/bread/{id}                     | voyager.bread.update              | TCG\Voyager\Http\Controllers\VoyagerBreadController@update             | web,admin.user                                       |
|        | DELETE    | admin/bread/{id}                     | voyager.bread.delete              | TCG\Voyager\Http\Controllers\VoyagerBreadController@destroy            | web,admin.user                                       |
|        | GET|HEAD  | admin/bread/{table}/create           | voyager.bread.create              | TCG\Voyager\Http\Controllers\VoyagerBreadController@create             | web,admin.user                                       |
|        | GET|HEAD  | admin/bread/{table}/edit             | voyager.bread.edit                | TCG\Voyager\Http\Controllers\VoyagerBreadController@edit               | web,admin.user                                       |

```

**Comment by raymondugv** [comment]
> Does that happen only after saving or also when you go to Tools -> BREAD?
> 
> How are your routes for bread?
> 
> ```
> php artisan route:list | grep bread
> 
> |        | POST      | admin/bread                          | voyager.bread.store               | TCG\Voyager\Http\Controllers\VoyagerBreadController@store              | web,admin.user                                       |
> |        | GET|HEAD  | admin/bread                          | voyager.bread.index               | TCG\Voyager\Http\Controllers\VoyagerBreadController@index              | web,admin.user                                       |
> |        | GET|HEAD  | admin/bread/delete_relationship/{id} | voyager.bread.delete_relationship | TCG\Voyager\Http\Controllers\VoyagerBreadController@deleteRelationship | web,admin.user                                       |
> |        | POST      | admin/bread/relationship             | voyager.bread.relationship        | TCG\Voyager\Http\Controllers\VoyagerBreadController@addRelationship    | web,admin.user                                       |
> |        | PUT       | admin/bread/{id}                     | voyager.bread.update              | TCG\Voyager\Http\Controllers\VoyagerBreadController@update             | web,admin.user                                       |
> |        | DELETE    | admin/bread/{id}                     | voyager.bread.delete              | TCG\Voyager\Http\Controllers\VoyagerBreadController@destroy            | web,admin.user                                       |
> |        | GET|HEAD  | admin/bread/{table}/create           | voyager.bread.create              | TCG\Voyager\Http\Controllers\VoyagerBreadController@create             | web,admin.user                                       |
> |        | GET|HEAD  | admin/bread/{table}/edit             | voyager.bread.edit                | TCG\Voyager\Http\Controllers\VoyagerBreadController@edit               | web,admin.user                                       |
> ```

Thanks for replying, 

It just only happens when I save my BREAD.
This is my routes of bread.
![image](https://user-images.githubusercontent.com/21214764/72807856-d9799a00-3c8a-11ea-99bf-da49675f8865.png)

**Comment by MrCrayon** [comment]
Try:
```
php artisan route:clear
```

If after that is not working I have no idea but my bet is some sort of caching, when you go to Tools -> BREAD that's the same url you are redirected to after saving.

**Comment by raymondugv** [comment]
Thank you for responding, 

I've tried your way, but it didn't work...

**Comment by MrCrayon** [comment]
Can't do anything else with the information I have.

You'll need to debug what's happening, try to put a `dd()` in:
`TCG\Voyager\Http\Controllers\VoyagerBreadController@store`
or update depending when the error happens, to be sure you are reaching the right controller, if you get there try to find out why 404 is returned.

**Comment by raymondugv** [comment]
> Can't do anything else with the information I have.
> 
> You'll need to debug what's happening, try to put a `dd()` in:
> `TCG\Voyager\Http\Controllers\VoyagerBreadController@store`
> or update depending when the error happens, to be sure you are reaching the right controller, if you get there try to find out why 404 is returned.

I see, 
Thank you for supporting.

Please keep this post open till we can find a solution for it 😌.

**Comment by MrCrayon** [comment]
Did you try to debug?

**Comment by raymondugv** [comment]
> Did you try to debug?

Hi, 
Sorry for late respoding, 

I've tried the dd() method, but it still return 404 :|.

**Comment by MrCrayon** [comment]
Then it's not reaching the right controller and you'll have to find why.
It might still be a cache problem or maybe you override controllers, can't tell from here.

**Comment by raymondugv** [comment]
> Then it's not reaching the right controller and you'll have to find why.
> It might still be a cache problem or maybe you override controllers, can't tell from here.

I see,

I guess will take me sometimes to find out what's the problem. But since the beginning of Voyager installation, I haven't change or override any Voyager basic files, except View.

That's weird!.

**Comment by MrCrayon** [comment]
Since your original issue was solved I'm closing this.

Please open e new issue if you still didn't solve the 404 after saving.
